tiatoolbox/utils/metrics.py

```diff
@@ -69,7 +69,9 @@ def f1_detection(true, pred, radius):
 
 
 def dice(gt_mask, pred_mask):
-    r"""This function computes `Sørensen–Dice coefficient
+    r"""Computes Sørensen-Dice coefficient.
```

Comment:
```suggestion
    r"""Compute the Sørensen-Dice coefficient.
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/6ae8838cdb92759eb927005fd350f505d95fba9f